src/renderers/dom/shared/utils/isCustomComponent.js

```diff
@@ -14,11 +14,11 @@
 var DOMNamespaces = require('DOMNamespaces');
 var HTML_NAMESPACE = DOMNamespaces.Namespaces.html;
 
-function isCustomComponent(tagName, props, namespace) {
+function isCustomComponent(tagName, props, domElement) {
   if (tagName.indexOf('-') >= 0 || typeof props.is === 'string') {
     // TODO: We always have a namespace with fiber. Drop the first
     // check when Stack is removed.
-    return namespace == null || namespace === HTML_NAMESPACE;
+    return domElement == null || domElement.namespaceURI === HTML_NAMESPACE;
```

Comment:
`domElement` wouldn't be `null`.

Reply:
This is only the case when the development hooks fire. UnknownDOMPropertyHook fires before mount; before a namespace is assigned to an element. Same deal with InvalidARIAPropertyHook.

Reply:
*For stack